.github/workflows/ci.yaml

```diff
@@ -118,9 +150,47 @@ jobs:
       run: |
         MLRUN_DOCKER_REGISTRY=ghcr.io/ \
         MLRUN_DOCKER_CACHE_FROM_TAG=${{ steps.docker_cache.outputs.tag }} \
-        MLRUN_PYTHON_VERSION=${{ matrix.python-version }} \
+        MLRUN_PYTHON_VERSION=3.9 \
+        UNIT_TEST_IGNORE_PATH=server/ \
         make test-dockerized
 
+  tests-server-3_9:
+    name: Run server Dockerized Tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up python 3.9
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.9
+        cache: pip
+
+    - name: Extract git branch
+      id: git_info
+      run: |
+        echo "branch=$(echo ${GITHUB_BASE_REF#refs/heads/})" >> $GITHUB_OUTPUT
+
+    - name: Resolve docker cache tag
+      id: docker_cache
+      run: |
+        export version_suffix=$(echo "${{ steps.git_info.outputs.branch }}" | grep -E "^[0-9]+\.[0-9]+\.x$" | tr -d '.')
+        export unstable_tag=$(if [ -z "$version_suffix" ]; then echo "unstable-cache"; else echo "unstable-cache-$version_suffix";fi)
+        echo "tag=$(echo $unstable_tag)" >> $GITHUB_OUTPUT
+
+    - name: Install automation scripts dependencies
+      run: pip install -r automation/requirements.txt
+
+    - name: Run Dockerized tests
+      #  including one test from /tests, due to ML-9563 issue.
+      run: |
+        MLRUN_DOCKER_REGISTRY=ghcr.io/ \
+        MLRUN_DOCKER_CACHE_FROM_TAG=${{ steps.docker_cache.outputs.tag }} \
+        MLRUN_PYTHON_VERSION=3.9 \
+        UNIT_TEST_PATH="tests/feature-store/test_common.py::test_parse_feature_string_with_dot_in_feature_set_name server/" \
```

Comment:
This looks like you chose some random test just to make it work but it doesn't address the root cause.
Can you explain exactly what is missing to make it work without running one client test? There must be some way to import the fixtures needed in the server from the client tests.